src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: $transition-duration;
```

Comment:
same here

Reply:
Thank you for your review! I've corrected this mistakes and I will keep it in mind in the future